src/Magnum/OvrIntegration/Session.h

```diff
@@ -49,22 +49,173 @@
 namespace Magnum { namespace OvrIntegration {
 
 /**
-@brief Swap texture set
+ * @brief A full pose (rigid body) configuration with first and second derivatives.
+ *
+ * Body refers to any object for which ovrPoseStatef is providing data.
+ * It can be the HMD, Touch controller, sensor or something else. The context
+ * depends on the usage of the struct.
+ */
+class MAGNUM_OVRINTEGRATION_EXPORT PoseState {
+public:
```

Comment:
Please indent also the `public:`/`private:` labels to be consistent with the rest of the codebase.